db/migrate/20150416231106_add_source_to_webhook.rb

```diff
@@ -0,0 +1,12 @@
+class AddSourceToWebhook < ActiveRecord::Migration
+  def up
+    add_column :webhooks, :source, :string, null: false, default: 'any_ci'
+    change_column_default :webhooks, :source, nil
```

Comment:
Fair enough, I guess I thought it just doesn't seem to make much sense to require the app to validate the value as well.